README.md

```diff
@@ -1,30 +1,142 @@
-# README
+# Hamlet Depoy - Executor Python
 
-This is a basic instruction on how to launch/test project locally in development environment.
+This is a python-based CLI for the Hamlet Deploy application. It acts as the primary interface to Hamlet Deploy, its Plugins and Extensions.
 
-### Dev env setup
+See https://docs.hamlet.io for more info on Hamlet Deploy
+
+### Installation
+
+```bash
+# Clone repository
+git clone https://github.com/hamlet-io/executor-python
+
+# install dependencies
+cd ./executor-python/hamlet-cli
+pip install -r ./requirements/prod.txt -r ./requirements/dev.txt
+
+# install
+make install
+```
+
+Hamlet Deploy CLI is now installed.
+
+```bash
+hamlet --help
+```
+
+### Configuration
+
+The Hamlet Deploy - Executor Python does not require specific configuration, however as it functions as a wrapper around other Hamlet Deploy parts, those must each be configured before using the CLI.
+
+See the [Hamlet Deploy docs](https://docs.hamlet.io/docs/hamletdeploy/software/cli) site for more information.
+
+### Update
+
+```bash
+# pull down changes
+cd ./executor-python
+git pull
+```
+
+With the latest files, re-run the installation steps to complete the update.
+
+### Contribute
+
+1. fork the repository
+2. update your changes in a feature branch
+3. Push to your fork's origin
+4. Create a Pull Request from your Fork to the Upstream master.
+
+```bash
+# fork the repository on Github.
+# https://github.com/hamlet-io/executor-python.git
+
+# add your upstream repo
+git remote add upstream https://github.com/hamlet-io/executor-python.git
+
+# create your working branch
+git checkout -b feat/my-feature-branch
+
+# make your changes / updates / fixes in as many commits as necessary.
+
+# rebase your work into only relevant commit messages
+# example: I've made 3 commits, but two of them are fixups.
+# git rebase -i HEAD~3
+# mark the fixups and save the commits
+git rebase -i HEAD~<number of commits since HEAD>
+
+# push your feature branch to your origin (the fork)
+git push --set-upstream origin feat/my-feature-branch
+
+# on Github, create a PR from your fork/feature-branch to upstream/master.
+# make sure you complete any Issue/PR templates provided.
+```
```

Comment:
Maybe this would be better in the .github repo. It has support for CONTRIBUTION file that will be made available in all repos

Reply:
See https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file

Reply:
Will set this up and remove from the latest lot of PRs I've done that include this.